### DISCUSSION TITLE
Draw 3D forms

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#1024] [Q&A] [answered]
Is there an example (I have not found one) for drawing 3D shapes, such a as a cube for instance? Or are overlay shapes only limited to 2D with the current implementation?

**Comment by devemux86** [comment, marked as answer]
Something like the 3D buildings and 3D POI layers, see the [ExtrusionsTest](https://github.com/mapsforge/vtm/blob/master/vtm-playground/src/org/oscim/test/ExtrusionsTest.java) and [MapsforgePoi3DTest](https://github.com/mapsforge/vtm/blob/master/vtm-playground/src/org/oscim/test/MapsforgePoi3DTest.java) examples.

**Reply by demiantres** [reply]
Thanks. Although I have not tested it yet, the "Poi3DLayer" layer looks promising.
